benchmark/run.js

```diff
@@ -30,11 +40,17 @@ if (benchmarks.length === 0) {
   child.on('message', function(data) {
     // Construct configuration string, " A=a, B=b, ..."
     let conf = '';
+    let confseparator = ' ';
+    let separator = ': ';
+    if (data.addOpts.format == 'csv') {
+      confseparator = ',';
+      separator = ',';
+    }
     for (const key of Object.keys(data.conf)) {
-      conf += ' ' + key + '=' + JSON.stringify(data.conf[key]);
+      conf += confseparator + key + '=' + JSON.stringify(data.conf[key]);
```

Comment:
This assumes that each benchmark has the same amount of parameters. You should do as in [compare.js#L71](https://github.com/nodejs/node/blob/master/benchmark/compare.js#L71) and just have `conf` being one field (remember to escape `"`).